src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/NeedsTest.kt

```diff
@@ -11,7 +11,7 @@ class NeedsTest : MockedEndpointsTestBase() {
 
   @Test
   fun `severe needs 18 points plus 2 OGRS make change level 3`() {
-    val crn = "X333444"
+    val crn = "X333445"
```

Comment:
This is a bit dodgy - if we reuse CRNs across integration tests then one of the tests fails. Should look into why that is really